### PR TITLE
[CELEBORN-822][DOC] Add quick start guide

### PR DESCRIPTION
**Pull request by waitinfuture**
### What changes were proposed in this pull request?
As title.
image

### Why are the changes needed?
As title.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
No.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -17,6 +16,97 @@ license: |
...
+===
+This documentation gives a quick start guide for running Spark with Celeborn.
 
-Apache Celeborn (Incubating)
-===
\ No newline at end of file
+### Download Celeborn
+Download the latest Celeborn binary from the [Downloading Page](https://celeborn.apache.org/download/).
+Decompress the binary into ```$CELEBORN_HOME```
+```angular2html
```
Comment (pan3793) [inline docs/README.md:26]: maybe `shell` or `bash`
```suggestion
```shell
```

File: docs/README.md [hunk 2]
```diff
@@ -17,6 +16,97 @@ license: |
...
+Quick Start
+===
+This documentation gives a quick start guide for running Spark with Celeborn.
 
-Apache Celeborn (Incubating)
-===
\ No newline at end of file
+### Download Celeborn
+Download the latest Celeborn binary from the [Downloading Page](https://celeborn.apache.org/download/).
+Decompress the binary into ```$CELEBORN_HOME```
```
Comment (pan3793) [inline docs/README.md:25]: Use single backtick for inline code

```suggestion
Decompress the binary into `$CELEBORN_HOME`
```

File: docs/README.md [hunk 3]
```diff
@@ -17,6 +16,97 @@ license: |
...
+```angular2html
+tar -C <DST_DIR> -zxvf apache-celeborn-<VERSION>-bin.tgz
+```
+
+## Configure Logging and Storage
+#### Configure Logging
+```angular2html
+cd $CELEBORN_HOME/conf
+mv log4j2.xml.template log4j2.xml
```
Comment (pan3793) [inline docs/README.md:34]: I suppose `cp` is encouraged rahter `mv`

File: docs/README.md (hunk 2 above)
Comment (pan3793) [reply docs/README.md:25]: seems we should guide user to `export CELEBORN_HOME` first

File: docs/README.md [hunk 4]
```diff
@@ -17,6 +16,97 @@ license: |
...
+```
+#### Configure Storage
+Configure the directory to store shuffle data, for example $CELEBORN_HOME/shuffle
```
Comment (pan3793) [inline docs/README.md:37]:
```suggestion
Configure the directory to store shuffle data, for example `$CELEBORN_HOME/shuffle`
```

File: docs/README.md [hunk 5]
```diff
@@ -17,6 +16,97 @@ license: |
...
+UserResourceConsumption: empty
+WorkerRef: null
+```
+
+## Start Spark with Celeborn
+#### Copy Celeborn Client to Spark's jars
+Celeborn release binary contains clients for Spark2.x and Spark3.x, copy the corresponding client jar into Spark's
+```jars/``` directory:
+```angular2html
+cp $CELEBORN_HOME/spark/${Celeborn Client Jar} $SPARK_HOME/jars/
```
Comment (pan3793) [inline docs/README.md:86]:
```suggestion
cp $CELEBORN_HOME/spark/<Celeborn Client Jar> $SPARK_HOME/jars/
```

File: docs/README.md [hunk 6]
```diff
@@ -17,6 +16,97 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+Quick Start
+===
+This documentation gives a quick start guide for running Spark with Celeborn.
```
Comment (pan3793) [inline docs/README.md:21]: When a project belongs to ASF first appears in a artical, we should call it "Apache XYZ"

```suggestion
This documentation gives a quick start guide for running Apache Spark with Apache Celeborn(Incubating).
```

File: docs/README.md [hunk 7]
```diff
@@ -17,6 +16,97 @@ license: |
...
+HeartbeatElapsedSeconds: xxx
+Disks:
+  DiskInfo0: xxx
+UserResourceConsumption: empty
+WorkerRef: null
+```
+
+## Start Spark with Celeborn
+#### Copy Celeborn Client to Spark's jars
+Celeborn release binary contains clients for Spark2.x and Spark3.x, copy the corresponding client jar into Spark's
```
Comment (cfmcgrady) [inline docs/README.md:83]:
```suggestion
Celeborn release binary contains clients for Spark 2.x and Spark 3.x, copy the corresponding client jar into Spark's
```

**Comment by waitinfuture** [comment]
@pan3793 updated, please take a look again

File: docs/README.md [hunk 8]
```diff
@@ -17,6 +16,98 @@ license: |
...
+
+## Start Spark with Celeborn
+#### Copy Celeborn Client to Spark's jars
+Celeborn release binary contains clients for Spark 2.x and Spark 3.x, copy the corresponding client jar into Spark's
+```jars/``` directory:
+```shell
+cp $CELEBORN_HOME/spark/<Celeborn Client Jar> $SPARK_HOME/jars/
+```
+#### Start spark-shell
+Set ```spark.shuffle.manager``` to Celeborn's ShuffleManager, and turn off ```spark.shuffle.service.enabled```:
```
Comment (pan3793) [inline docs/README.md:90]:
```suggestion
Set `spark.shuffle.manager` to Celeborn's ShuffleManager, and turn off `spark.shuffle.service.enabled`:
```

File: docs/README.md [hunk 9]
```diff
@@ -17,6 +16,98 @@ license: |
...
+echo "celeborn.worker.storage.dirs=$CELEBORN_HOME/shuffle" > celeborn-defaults.conf
+```
+
+## Start Celeborn Service
+#### Start Master
+```shell
+cd $CELEBORN_HOME
+./sbin/start-master.sh
+```
+You should see ```Master```'s ip:port in the log:
```
Comment (pan3793) [inline docs/README.md:50]:
```suggestion
You should see `Master`'s ip:port in the log:
```

**Review by pan3793 — APPROVED** [review]

**Comment by waitinfuture** [comment]
Thanks, merging to main.

**Pull request closed without merging**